frontend/cypress/integration/utils/login.js

```diff
@@ -22,7 +22,7 @@ const loginAsDefault = (duration = 3600) => {
 };
 
 const logout = () => {
-  cy.get('[data-test-id="logout-button"]').click();
+  cy.get('[data-test-id="logout-button"]').click({ force: true });
```

Comment:
Why did you add `force: true` here?

Reply:
because it can be hidden by the convention popup